src/foam/u2/detail/ChoiceSectionDetailView.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.detail',
+  name: 'ChoiceSectionDetailView',
+  extends: 'foam.u2.detail.AbstractSectionedDetailView',
+
+  requires: [
+    'foam.core.ArraySlot',
+    'foam.u2.detail.SectionView',
+    'foam.u2.view.ChoiceView'
+  ],
+
+  css: `
+    ^ .foam-u2-view-ScrollTableView table {
+      width: 100%;
+    }
+
+    ^ .foam-u2-tag-Select {
+        text-align-last:center;
+      }
+
+    ^ .foam-u2-tag-Select:after {
+        text-align-last:center;
+      }
+
+      .choicePosition {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        text-align: center;
+      }
+  
+      .action-button {
+        border-radius: 10px;
+        font-size: 20px;
+        font-weight: bold;
+        padding: 5px;
+        width: 100px;
+      }
+  `,
+
+  properties: [
+    {
+      name: 'currentIndex',
+      preSet: function(o,n) {
+        if ( n < 1 ) return 0;
+        if ( n >= this.newSections.length ) return this.newSections.length - 1;
+        return n;
+      },
+      value: 0
+    },
+    {
+      name: 'newSections'
```

Comment:
```suggestion
      name: 'visibleSections'
```

I think this is a more meaningful name.